templates/security/cve/_cve-table.html

```diff
@@ -98,6 +98,40 @@
                   {% endif %}
                 </small>
               </div>
+              <div class="icon-container__icon icon-cve-info">
+                {% if statuses[release.codename].status != 'DNE'%}
+                  <i class="p-icon--information"></i>
+                {% endif %}
+                <small class="cve-tooltip">
+                  <div class="arrow-up back"></div>
+                  <div class="arrow-up front"></div>
+                  {% if statuses[release.codename].status != 'DNE' and statuses[release.codename].status != 'not-affected' %}
+                    {%if cve.priority == "critical" %}
+                      <b>Critical:</b> Open vulnerability that is a world-burning problem and is exploitable for most Ubuntu users. Examples include remote root privilege escalations or remote data theft.
```

Comment:
```suggestion
                      <b>Critical:</b> A world-burning problem that is exploitable for most Ubuntu users. Examples include remote root privilege escalations or remote data theft.
```
Now that we’re showing status and priority separately, it’s no longer true that CVEs with a priority are always “open”. (Sorry I didn’t notice this until now.)

Reply:
Done.